### PR TITLE
Add ipv6 blst

### PR DESCRIPTION
**Pull request by andir**
fixes issue #7

### CODE REVIEW

**Comment by Adam-** [comment]
I think this should maybe be more like type = ipv4, ipv6; since blacklists can support either/or. I don't want to have to add two blacklist{} blocks for the same blacklist just for two different address families. Also I think there's probably nothing wrong with sending queries for ipv6 addresses to dnsbls which don't expect them, they should return NXDOMAIN.

File: src/misc.c [hunk 1]
```diff
@@ -95,3 +95,70 @@ stripws(char *txt)
...
+
+const int 
+format_reverse_inet(const struct in_addr* addr, char *buf, size_t buf_size, const char * suffix) {
+  unsigned char a, b, c, d;
+  d = (unsigned char)(addr->s_addr >> 24) & 0xFF;
+  c = (unsigned char)(addr->s_addr >> 16) & 0xFF;
+  b = (unsigned char)(addr->s_addr >>  8) & 0xFF;
+  a = (unsigned char) addr->s_addr & 0xFF;
+#ifdef WORDS_BIGENDIAN
+  snprintf(buf, sizeof(buf), "%d.%d.%d.%d.%s", a, b, c, d, suffix);
```
Comment (Adam-) [inline src/misc.c]: This should be buf_size

Comment (andir) [reply src/misc.c]: fixed in newer version

**Comment by andir** [comment]
I added your commend but I have a rather strong opinion about not spamming blacklists that do not support ipv4 or ipv6 with either type of requests. Would you think that is a fine behaviour?

**Comment by andir** [comment]
After looking at it again the BISON generated files do not really do what I wanted them to do.. I'll dig into that during the day.

**Comment by miwob** [comment]
Thanks for your contribution. Proper ipv6 support has been implemented using some pieces of your initial patch.

**Pull request closed without merging**